packages/core/src/App/Containers/RealAccountSignup/personal-details-form.js

```diff
@@ -94,19 +94,23 @@ export const personalDetailsConfig = ({ can_upgrade_to }) => {
             ],
             salutation_list: [
                 {
-                    text: localize('Mr'),
+                    id: 'Mr',
+                    label: localize('Mr'),
                     value: 'Mr',
                 },
                 {
-                    text: localize('Ms'),
+                    id: 'Ms',
+                    label: localize('Ms'),
                     value: 'Ms',
                 },
                 {
-                    text: localize('Mrs'),
+                    id: 'Mrs',
+                    label: localize('Mrs'),
                     value: 'Mrs',
                 },
```

Comment:
Can we use just either `id` or `value` as their values are the same?

Reply:
Please read my response to Mamali and correct me if I am mistaken about the id.